selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java

```diff
@@ -66,38 +64,40 @@
           + " at org.junit.Assert.assertTrue(Assert.java:41)\n"
           + " at org.junit.Assert.assertFalse(Assert.java:64)\n"
           + " at org.junit.Assert.assertFalse(Assert.java:74)\n"
-          + " at org.eclipse.che.examples.AppAnotherTest.shouldFailOfAppAnother(AppAnotherTest.java:33)";
+          + " at org.eclipse.che.examples.AppAnotherTest.shouldFailOfAppAnother(AppAnotherTest.java:34)";
+
   @Inject private JavaTestRunnerPluginConsole pluginConsole;
   @Inject private ProjectExplorer projectExplorer;
   @Inject private Loader loader;
   @Inject private NotificationsPopupPanel notifications;
   @Inject private Menu menu;
 
-  @InjectTestWorkspace(template = WorkspaceTemplate.CODENVY_UBUNTU_JDK8)
-  private TestWorkspace ws;
+  @Inject private TestWorkspace ws;
 
   @Inject private Ide ide;
   @Inject private Consoles consoles;
   @Inject private CodenvyEditor editor;
-  @Inject private TestProjectServiceClient testProjectServiceClient;
   @Inject private TestCommandServiceClient testCommandServiceClient;
   @Inject private CommandsPalette commandsPalette;
+  @Inject private TestProjectServiceClient projectServiceClient;
 
   @BeforeClass
   public void prepareTestProject() throws Exception {
     CompileCommand compileCommand = new CompileCommand();
     testCommandServiceClient.createCommand(DtoConverter.asDto(compileCommand), ws.getId());
-
-    URL resource = getClass().getResource("/projects/plugins/JavaTestRunnerPlugin/junit4-tests");
-    testProjectServiceClient.importProject(
+    projectServiceClient.importProject(
         ws.getId(),
-        Paths.get(resource.toURI()),
```

Comment:
Consider porting [this class](https://github.com/eclipse/che/blob/spi/selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/WorkspaceDtoDeserializer.java) to master to avoid painful merge conflicts.